4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4896 @@
+{
```

Comment:
<blockquote>Oops theres an error in the model specification, did you catch it? We accidentally flipped the input parameters to our binomial likelihood. Good thing we caught it early.</blockquote><p>I don't see the error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>